src/generate-missing-collectible-info.ts

```diff
@@ -26,6 +26,9 @@ const nonCollectibleItems = inventoryItems.filter((item) => !item.collectibleHas
 const collectibleItems = inventoryItems.filter((item) => item.collectibleHash);
 
 collectibleItems.forEach((collectibleItem) => {
+  if (!collectibleItem.displayProperties?.name) {
+    return;
+  }
```

Comment:
Some bad collectibles with no name -- this would otherwise pull in a bunch of dummy shaders in "source:eververse".